src/main/java/org/broadinstitute/hellbender/engine/FeatureDataSource.java

```diff
@@ -473,8 +473,8 @@ final void printCacheStats() {
     public SAMSequenceDictionary getSequenceDictionary() {
         SAMSequenceDictionary dict = null;
         final Object header = getHeader();
-        if ( header instanceof FeaturesHeader ) {
-            dict = ((FeaturesHeader)header).getDictionary();
+        if ( header instanceof SVFeaturesHeader ) {
```

Comment:
What's the rationale behind the `FeaturesHeader` -> `SVFeaturesHeader` migration?

Reply:
Why FeaturesHeader --> SVFeaturesHeader?  It just seemed to me that the name was too generic -- suggesting that it applied to all Features -- and, in particular, that it could easily be confused with the FeatureCodecHeader. I just wanted to make the name suggest to which Feature sub-types it applied.